tests/unit/form-widget.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { shallowMount } from '@vue/test-utils';
```

Comment:
You could move this squash this commit with the one introducing the `FormWidget` mixin